### PR TITLE
Exclude NonApplicants in Medicaid Gap Filling

### PR DESCRIPTION
**Pull request by saikumar9**
ME-180762646

### CODE REVIEW

**Comment by markgoho** [comment]
@saikumar9 it looks like you're updating aca_entities to a newer ref than what's being done in another open PR (#109) -- I'd recommend closing that other PR with this PR as a reference

File: spec/operations/eligibilities/determine_full_eligibility_spec.rb [hunk 1]
```diff
@@ -35,6 +35,10 @@
 
   # Dwayne is UQHP eligible and eligible for non_magi_reasons
   context 'cms simle test_case_a' do
+    before do
+      allow(Date).to receive(:today).and_return(Date.new(2021, 12, 1))
```
Comment (markgoho) [inline spec/operations/eligibilities/determine_full_eligibility_spec.rb]: why are we hard-coding dates like this?

**Review by markgoho — CHANGES_REQUESTED** [review]
there look to be a number of hard-coded dates, can we use something generalized instead?

File: spec/operations/eligibilities/determine_full_eligibility_spec.rb (hunk 1 above)
Comment (saikumar9) [reply spec/operations/eligibilities/determine_full_eligibility_spec.rb]: There are specific tests in this file that are meant to be run in the context of the year 2021 to compare FPL levels and others.

Comment (markgoho) [reply spec/operations/eligibilities/determine_full_eligibility_spec.rb]: but it's 2022 now, what's the purpose of keeping these tests?

Comment (saikumar9) [reply spec/operations/eligibilities/determine_full_eligibility_spec.rb]: Retrospective case.

Comment (markgoho) [reply spec/operations/eligibilities/determine_full_eligibility_spec.rb]: In what scenario would this test fail? E.g. code change, date change, etc.

**Review by Battula — APPROVED** [review]

**Review by markgoho — APPROVED** [review]

**Pull request merged**